src/modules/core/components/Lightbox.js

```diff
@@ -103,7 +99,7 @@ const Lightbox = ({
           <div className={classes.headerTitle}>{title}</div>
         </div>
         <div className={classes.contentContainer}>
-          <button onClick={closeLightbox} className={classes.closeButton} />
+          <button onClick={onClose} className={classes.closeButton} />
```

Comment:
I'd call this `closeBox` or something similar. Minor distinction but we're listening for a close event, but triggering one ourselves.

Reply:
But isn't the name from the component consumer's perspective? We want to run code when the lightbox is closed so I feel like onClose makes sense.

Reply:
Does `onClose` run when the box is closed, or does it close the box?